packages/util-rxjs/test/coldObservableProvider.test.ts

```diff
@@ -8,11 +9,24 @@ jest.mock('backoff-rxjs', () => ({
   retryBackoff: jest.fn().mockImplementation((...args) => jest.requireActual('backoff-rxjs').retryBackoff(...args))
 }));
 
+const createMockLogger = (): Logger => ({
```

Comment:
There's a test logger you can use in `util-dev`. One usage example is in [ogmios](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/ogmios/test/util.test.ts#L27) tests

Reply:
done here: 867945df0b58d37b9a1955b794e8950cfcf5b743